assets/src/edit-story/components/canvas/useCanvasKeys.js

```diff
@@ -113,6 +113,18 @@ function useCanvasKeys(ref) {
             currentSelectedIds?.length === 1
               ? getNodeForElement(currentSelectedIds[0])
               : null;
+
+          // Check if the selection event happend outside the canvas container,
+          // i.e Checklist panel, Document panel... etc. If there is any text
+          // selection, we should not add the preventScroll setting, so that
+          // user can select the text properly.
+          const sel = global.getSelection();
+          if (sel.anchorNode && sel.anchorOffset !== 0) {
```

Comment:
I think `sel.anchorNode` will always return an element. So I don't think this works.

If I run `window.getSelection().anchorNode` in dev tools it's  never `null`/`undefined`